test/bin/ci_phase_iso_build.sh

```diff
@@ -102,7 +102,7 @@ run_image_build() {
 
 # Run bootc image build
 run_bootc_image_build() {
-    $(dry_run) bash -x ./bin/build_bootc_images.sh -l ./image-blueprints/layer5-bootc
+    $(dry_run) env WITH_FLANNEL=1 bash -x ./bin/build_bootc_images.sh -l ./image-blueprints/layer5-bootc
```

Comment:
I'm sorry, but I suggested a wrong place for this setting. This one is passed to bootc container builds, where the flag is not used. 

Instead, we need to enable the flag when building RPMs in CI on [this line](https://github.com/openshift/microshift/blob/main/test/bin/ci_phase_iso_build.sh#L128).

Reply:
@ggiguash  Thanks for the pointer and looks like now it is working.

Reply:
So from where we can check scenario is working https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_microshift/4034/pull-ci-openshift-microshift-main-e2e-aws-tests-bootc-periodic/1843889678214762496/artifacts/e2e-aws-tests-bootc-periodic/openshift-microshift-e2e-metal-tests/artifacts/scenario-info/ doesn't show flannel one.

Reply:
It is part of https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_microshift/4034/pull-ci-openshift-microshift-main-e2e-aws-tests-periodic/1843889683604443136/artifacts/e2e-aws-tests-periodic/openshift-microshift-e2e-metal-tests/artifacts/scenario-info/ but this job failed, how to debug it?